package-lock.json

```diff
@@ -23897,6 +23898,12 @@
         "node": ">=10"
       }
     },
+    "node_modules/prebuild-install/node_modules/napi-build-utils": {
```

Comment:
Not a huge deal, but it looks like we now have duplication of `prebuild-install` and `napi-build-utils` because of the different versions being requested by kerberos and mongodb-client-encryption. Might be a good idea to dedupe them if possible.

Reply:
As long as it's a pinky promise, we can let this one slide 😆